compute_tools/src/spec.rs

```diff
@@ -380,5 +382,20 @@ pub fn handle_grants(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
         client.execute(query.as_str(), &[])?;
     }
 
+    // Do some per-database access adjustments
+    for db in &node.spec.cluster.databases {
+        let mut db_connstr = node.connstr.clone();
```

Comment:
I wonder if that could be created before the loop once, and then `set_path` could be called on the same data.

Reply:
Changed, thanks